apps/web/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Listbox,
+  ListboxButton,
+  ListboxOptions,
+  Transition,
+} from "@headlessui/react";
+import { ChevronUpDownIcon } from "@heroicons/react/24/outline";
+
+import { Option } from "./components/Option";
+
+export interface Option {
+  value: string | number;
+  label: string;
```

Comment:
Also, let's make it optional and render the `value` when it's not set

Reply:
Done :+1: